src/Query/Conditions/Interface/LikeConditionInterface.php

```diff
@@ -5,30 +5,30 @@
 namespace Yiisoft\Db\Query\Conditions\Interface;
 
 use Iterator;
+use Yiisoft\Db\Expression\Expression;
 use Yiisoft\Db\Expression\ExpressionInterface;
 
 interface LikeConditionInterface extends ConditionInterface
 {
     /**
-     * @psalm-return string|string[]|ExpressionInterface The column name. If it is an array, a composite `IN` condition
-     * will be generated.
+     * @psalm-return string|Expression The column name.
      */
-    public function getColumn(): string|array|ExpressionInterface;
+    public function getColumn(): string|Expression;
 
```

Comment:
ExpressionInterface?

Reply:
The interface is only used to check that it is used but it does not have any methods, because the conditions have different methods, for example only `Expression:class` has to_string().